mutmut/cache.py

```diff
@@ -136,6 +139,51 @@ def print_stuff(title, query):
     print_stuff('Untested', select(x for x in Mutant if x.status == UNTESTED))
 
 
+def get_unified_diff(argument, dict_synonyms):
```

Comment:
Refactoring the code that was performing the unified diff on `__main__.py`, and bringing it here so that we can also use it for the JUnit report